src/main/scala/io/iohk/ethereum/testmode/TestLedgerWrapper.scala

```diff
@@ -0,0 +1,19 @@
+package io.iohk.ethereum.testmode
+
+import io.iohk.ethereum.consensus.Consensus
+import io.iohk.ethereum.domain.BlockchainImpl
+import io.iohk.ethereum.ledger.{Ledger, LedgerImpl}
+import io.iohk.ethereum.utils.BlockchainConfig
+import io.iohk.ethereum.utils.Config.SyncConfig
+
+/**
+  * Provides a ledger instance with modifiable blockchain config (used in testmode).
+  */
+class TestLedgerWrapper(
+    blockchain: BlockchainImpl,
+    syncConfig: SyncConfig,
+    consensus: Consensus,
+    var blockchainConfig: BlockchainConfig) { // var as it's modifiable by test_ RPC endpoints
```

Comment:
Couldn't we just create a new instance of this class?

Reply:
Not really. It's a bit tricky, because Ledger is being passed as a parameter to various other components on initialization (see NodeBuilder.scala). I wanted to create new instances, but I didn't find an elegant way to update those classes with new ledger (without introducing a var elsewhere).